DeusEx/Classes/DXRFixup.uc

```diff
@@ -1,12 +1,30 @@
 class DXRFixup expands DXRBase;
 
 function FirstEntry()
+{
+    Super.FirstEntry();
+
+    Level.AmbientBrightness += dxr.flags.brightness;
+
+    if( dxr.dxInfo.missionNumber == 6 )
```

Comment:
While we only have one check right now, I still vastly prefer switch statements, that's just personal preference though.

Reply:
I think since these are just single-line if statements, I like them either way really, and an if statement can give you a little more freedom sometimes